bower.json

```diff
@@ -17,8 +17,9 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "purescript-base": "^0.2.0",
     "purescript-aff": "^0.11.3",
+    "purescript-aff-timeout": "purescript-aff-reattempt#*",
```

Comment:
What's going on here? :smile:

Also if we're not referring to a released version we should reference a specific commit hash at least - when you use `*` or a branch name bower will locally cache the first thing it downloads from there. The problem with that is people can end up with different versions cached locally if they `bower install` at different points in time.